src/main.cc

```diff
@@ -25,6 +25,7 @@
 #include "gltf_loader.h"
 #include "gpl/solar_position.h"
 #include "types.h"
+#include<CLI/CLI.hpp>
```

Comment:
can you move this up with the other third-party includes (between `<bvh/*>` and `<glm/*>`)? also, missed a space:
```suggestion
#include <CLI/CLI.hpp>
```